src/version.cr

```diff
@@ -0,0 +1,3 @@
+module Mint
+  VERSION = "0.8.0-devel"
```

Comment:
```suggestion
  VERSION = {{ `shards version #{__DIR__}`.chomp.stringify }}
```

Reply:
Sometimes Mint gets a new version released without the updated version in the CLI, we do not want to make this problem worse. As @Sija said, one place to define the version is a good idea.

Reply:
@s0kil could you give me an use case for when that could happen? I'm fine with using the suggested compile-time macro by @Sija if that's the better choice. @Sija will that work cross-platform?

Reply:
@Sija I was thinking of using the [`read_file`](https://crystal-lang.org/api/0.33.0/Crystal/Macros.html#read_file(filename):StringLiteral-instance-method) macro, is `shards version` better?

Reply:
@gdotdesign `shards version` exists for this exact reason, what's the point of using `read_file` instead?

@KevinSjoberg yes, it is (as long `shards` is).

Reply:
I guess my only concern if it will work on windows builds.

Reply:
It should, of course. If it ain't, that would mean there is a bigger problem than that (`shards`). Also, Windows support is still in its infancy, we can always fix things l8r if deemed necessary.

Reply:
@KevinSjoberg https://github.com/mint-lang/mint/issues/147

Reply:
@s0kil Ah, shit happens. 🙂